docs/pages/forge_visualization.md

```diff
@@ -1,160 +1 @@
-Visualizing af::array with Forge {#forge_visualization}
```

Comment:
Is this correct? You deleted the entire text but one line (https://github.com/bkloppenborg/arrayfire/blob/doc_improvements/docs/pages/forge_visualization.md)